FluentUI/src/main/res/values-night/themes.xml

```diff
@@ -63,11 +63,11 @@
         <!--ContextualCommandBar-->
         <item name="fluentuiContextualCommandBarBackgroundColor">@color/fluentui_gray_600</item>
         <item name="fluentuiContextualCommandBarBackgroundColorPressed">@color/fluentui_gray_900</item>
-        <item name="fluentuiContextualCommandBarBackgroundColorSelected">#0086F0</item>
+        <item name="fluentuiContextualCommandBarBackgroundColorSelected">@color/fluentui_communication_blue</item>
         <item name="fluentuiContextualCommandBarIconTint">@color/fluentui_gray_100</item>
         <item name="fluentuiContextualCommandBarIconTintDisabled">@color/fluentui_gray_400</item>
         <item name="fluentuiContextualCommandBarIconTintSelected">@color/fluentui_black</item>
-        <item name="fluentuiContextualCommandBarDismissBackgroundColor">#000000</item>
+        <item name="fluentuiContextualCommandBarDismissBackgroundColor">@color/black</item>
```

Comment:
This causes a compilation/link error for me when pulling in the library, it should be @color/fluentui_black I think?